plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1840,6 +1862,11 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
             boolean changeDataFeedEnabled = (Boolean) properties.get(CHANGE_DATA_FEED_ENABLED_PROPERTY)
                     .orElseThrow(() -> new IllegalArgumentException("The change_data_feed_enabled property cannot be empty"));
             if (changeDataFeedEnabled) {
+                Set<String> columnNames = getColumns(handle.getMetadataEntry()).stream().map(DeltaLakeColumnHandle::getName).collect(toImmutableSet());
+                Set<String> conflicts = Sets.intersection(columnNames, CHANGE_DATA_FEED_COLUMN_NAMES);
+                if (!conflicts.isEmpty()) {
+                    throw new TrinoException(NOT_SUPPORTED, "Unable to enable change data feed when the table contains %s columns: %s".formatted(CHANGE_DATA_FEED_COLUMN_NAMES, conflicts));
```

Comment:
no need to provide CHANGE_DATA_FEED_COLUMN_NAMES in the message

`"Unable to enable change data feed because table contains %s columns".formatted(conflicts)`